lya/data.py

```diff
@@ -2,6 +2,7 @@
 from .theory import get_lcdm_pk
 import numpy as np
 import matplotlib.pyplot as plt
+from theory import get_theory_pk
```

Comment:
this should be `.theory` (or have the first line be `from .theory import get_lcdm_pk, get_theory_pk`)